WordPress/Classes/ViewRelated/Reader/ReaderMenuViewController.swift

```diff
@@ -0,0 +1,172 @@
+import Foundation
+import WordPressShared
+
+
+/// The menu for the reader.
+///
+@objc class ReaderMenuViewController : UITableViewController
+{
+    let cellIdentifier = "MenuCellIdentifier"
+
+    lazy var viewModel: ReaderMenuViewModel = {
+        let vm = ReaderMenuViewModel()
+        vm.delegate = self
+        return vm
+    }()
+
+
+    /// A convenience method for instantiating the controller.
+    ///
+    /// - Returns: An instance of the controller.
+    ///
+    class func controller() -> ReaderMenuViewController {
+        return ReaderMenuViewController(style: .Grouped)
+    }
+
+
+    // MARK: - Lifecycle Methods
+
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationItem.title = NSLocalizedString("Reader", comment: "")
```

Comment:
There actually is tho I haven't dug into it to understand the particulars.  I found that setting `title` will cause the text to appear at the bottom of the reader's tab in the tab bar.  Setting `navigationItem.title` doesn't.